hydradx/notebooks/Omnipool/LP_returns_analysis2.ipynb

```diff
@@ -0,0 +1,437 @@
wIozLQvGY7EiJD0R0dyEs+gUMARFVz4OFOn5G7czvH8sMK6x0Dj1Qx+vFIAWbMsFRGMdGjh4hB8+dL6VyABg2kb79+0LIlrFoFlSpJytby5bKcj/NIKV/I88/LWAqFQqFQXCsMBqhdG06dsm3Pyro287lKOPhVAiDj+H7OfTgB1watcapWF9e6haOcHCpVxd7DG3svP3JiL9is0/PyOPpYW9KP7rFpD32wMZqTC14d+165g1AoFIobnJDnpuNSuwnuze8AOzsO3VvDvC76x4+xc/ei8tNvXrsJXoKL384wP0/4YylZ507hVKV2CVsoKhoV8XOdsuviagAaH4Lm+6HhUTFVrnUGap+G2qegyWFJ2apzovD21c9KP/9YqHQRmh6EVnug3gmocxI0zYCrgzcNQ6F1eiNC6nTB3smNmDM7yM1MIS9ZPHUCY6Dp7hxOfTWZY/+I6FP7JHikyn7cvatQ5aK9JcqnII0by13S8+fFJPPIESmpazTC7t0QHCwn0y+8YNnmyy9hxAjLcv/+l/16KhQKhUJx2fzzj3jMXbgAmZkStZqQcK1ndUUxRfxEfDCOhA1LOD97EqdfGEDWOYsAVvuDldSe9Ys5WkM35pH8n8WkOTPsKHva2RcSfUzoWRk4BqqIH4VCoSgOTdPw6zMMpyq1cAquTp1P1uJ1Zz/z+qhvZ5CXkXYNZ1g8xswMYld8CUCdT9YCkHpg27WcUrHk5eXRsmVL+vYt/mbE8uXL0TSNgkWrwsPDcXd3Z9asWVd6muVCRfxcR+xI/oNm7h1wMbhyOmYHOEPzfeCdDBcqQ5q79Ev2gAQfqBYO6a4QHShVtnQNkjzBJxEccqDBUXDMhmRPOFkXPOx98Tgbj38stNxnBBJlwCOhBB0JJQjwHPos2akJZDkD69ZxpnImcS7p+AQ3ompwW7Lnf45dUgpOlaoT2KQbATO+QWvSpHifBBOVK5e8vl07EYbs7EQIGjlSooWcnGSdQqFQKBTXmkqV4C4rHxo/v5te+DEVXMgKt73LlHbIksrl3flem3V5yQnkJSeQEx+Ng28giX+ttFnf6r88NIOBc59MMt8FdggsQ+qdQqFQ3OJ4deiFV4deZJ07Rdb5M5wY3ZPoHz4i+ImXK2wfObGR2PtWQjOUP1YkO/o8Ya8NA0BzcsallmSI6NmZJW12zfj4449p2LAhyclFm8ympKTw8ccf066I69MJEybQu3fvKz3FcqMifq4iRt2Iruvmilw5eg6fnn+Z1rs1Wu/WePZET/ocqMprZx7jb5edOGdKlE7Tg9DgmKRntdoNjQ9LxE/dE1DlnAhAwZH50T3R4BsP9Y+BUzY0CoXKF2TbejviCY6CmmfA0c0XFzd/GhyBBq6tcKkhETvJ388l89cf0X/6ET0xnqpH06l/xomWTrdj/HQ2dkkpOBvcaLDuLIEzvkEDKXlbETg4iOhjYscO2LJFzDUVCoVCobjecHMrujLYTYS9ly+VR71lXq42ZR4g5bgB6n5avAdfxKyx6Lm5RP/0CZqDI3Xn/kHrXbr5IiJ45Kvmvm5N1E0ehUKhKCtOVWrj0boLAHGrvy2yT/LOTexuo7G7jcaZV4cRs/xzdF0nNymei4s+IPbX+aQf32/un7L3b/Z18eZAr8qcfWskAGlHdpObGFfqeSX9+zuRX7/DwXuqkLLrT1zqtaDl1nQ0RycAjNnFp0nreXlkR0UUu/5y0HVdrsdTkwqtO3fuHKtXr2bkyJHFbv/qq68yadIknJ2dbdp/+eUXatasSeOC1ifXEeqK+gqTlBvPvpStdPa+l+57/UjWEwFo5nY7DjiwO22Lbf+8eFbHy5e21mkImjAdp/Ao/D/6CB3x9IkMhoAYqaTllSypXPa5oFtpJsEXoHKkPK8RBqdrgbe9Px5nYvGLhaaHrcqj/72HRgYDeuUQ4sYMISshEmNeLv4Xsjh5fgt28XFc/G4mztXqEdisBwGvz7Fsu369VDdRKBQKheJWw9kZ4kp/InyjEjzyFdxb3IExKxOP1ncS/u4okrasBDs73Jt3KHa7hPWLSVi/GIBa7y3Ds213m/V2ru602p5N5tnjpa4AViQXL8LevdCrV/nHKCuhoRKdPHs2tG599farUCgUBdDs7Ql46DlifpqNnpuDZu9AxulQkndswDGoGqdfGGDuG792EfFrF5G6byvxaxfZjNNo8QFc6jQlZecmszAS99s3uDfvwNm3n8TBP5hqL3+BY0AIrg1aFjufiPfHE/3jRzZtVSd8gKZpGBxFMDGmp3BibB8qDZ1Q6H/DidE9Sdn1J3U+Wo1Xx3vK9Froubmk7v8H5xoNiFn2GdlRZ6n87Ds4+AcTPu1pUvdtxb1lZ2J//pzKz75D8IiXzNuOGzeOGTNmkJKSUuTYe/bsISIigj59+jBz5kxze2pqKu+99x4bNmy4btO8QAk/V5zH97cknPBC7QfSbPMaG4VCtiOcqQF+cRB0EVruAcdJQ6FKFZgxAy08HJdt26j+3jSCDx/BJROMGlysBG7pkGeQtC7PZDAYRfypdFHSxJyzAGIBqHEWqbiVmQkffgiJiTB0KNq58/hPnmkzr1quEN0kkOCPFuM84xMwiT6//CIlbr29K/gVUygUCoXiBsHJCXbuhDFjIDkZ+vaVtOW9e+Hpp6FevWs9wwrDo03XQm0utZticHYt1O5//0hSD24n89Qhc5tXxz5FjqvZO+BS+zLukGZnQ1CQPI+IkHOmK01CgqWgRZs28NVXUtV09GhJB1QoFIqrjFvjtsQgnmqpB7YRPu3pQn3qzllH8n8bufjtDOLXLsLeLwiPlp1I3rGBvJREIr9+h5pvfUfkF28AUG3yZ4RPf4azbz8JSOrXqfGS2lvjrUX4dB+EIT+Cx4Su60QvketFp+r1CRo2Ef/7LRE0mpMIPym7/iR5+3qS/1mDX78nqD7lMzR7B3KTE0jZ9ScAJ8f1waNtD6q/8iUOAZWLLAGf+PcqTo2/F79+T1DthU848+owEv
```

Comment:
I think this is overestimating trade volume. We don't have to make a claim on how much we have seen though, I think we can just provide the 0.3% to 10% range without further comment, they can look on our app to see what the last 24hr trade volume is for each asset.